packages/vue-client/src/components/boards/BoardPreview.vue

```diff
@@ -0,0 +1,117 @@
+<script setup lang="ts">
```

Comment:
There is some code duplicated from other board components here.

Reply:
Wait, why copy the code?  This feels like something we can just use [`DefaultBoard`](https://github.com/govariantsteam/govariants/blob/020709a8145b2fa7c66e5327b1379433cf0de587/packages/vue-client/src/components/boards/DefaultBoard.vue) for

Reply:
Hm good point, we could wrap it with `pointer-events: none` to prevent the hover effects, and get star points etc. for free.